src/profiles/drive.js

```diff
@@ -8,18 +8,51 @@ function getPresentationFolder(presentation) {
   return presentationFile.getParents().next();
 }
 
+function getTargetFolderName() {
+  return PropertiesService.getScriptProperties().getProperty('TARGET_FOLDER_NAME');
+}
+
 function getDriveStorageFolder() {
   const presentationFolder = getPresentationFolder(SlidesApp.getActivePresentation());
   const thumbnailsFolder = createThumbnailsFolder(presentationFolder, getTargetFolderName());
   
   return thumbnailsFolder;
 }
 
-
 function getLastUpdatedTime(docid) {
-  return DriveApp.getFileById(docid).getLastUpdated();
+  if(!docid) {
+    return 0;
+  }
+  try {
+    return DriveApp.getFileById(docid).getLastUpdated();
+  } catch(e) {
+    return 0;
+  }
 };
 
 function getCreatedDateTime(docid) {
   return DriveApp.getFileById(docid).getDateCreated();
-};
\ No newline at end of file
+};
+
+function trashFile(folder, file) {
+  folder.removeFile(file)
+  DriveApp.removeFile(file);
+  file.setTrashed(true);
+}
+
+function trashFileById(folder, fileId) {
+  const file = DriveApp.getFileById(fileId);
+  folder.removeFile(file)
```

Comment:
reuse trashFile() function instead of line 45, 46, 47

Reply:
fixed